src/main/java/net/snowflake/client/jdbc/SnowflakeChunkDownloader.java

```diff
@@ -685,7 +704,7 @@ private void waitForChunkReady(SnowflakeResultChunk currentChunk) throws Interru
                     authTimeout,
                     socketTimeout,
                     session));
-        downloaderFutures.put(nextChunkToDownload, downloaderFuture);
+        downloaderFutures.put(nextChunkToConsume, downloaderFuture);
```

Comment:
This is a typo. Since downloaderFutures is only used for cleaning up threads when the downloader is shutdown, this was not causing a problem.